main.cpp

```diff
@@ -331,25 +399,29 @@ int handle_args(int argc, char** argv) {
 
         if (vm.count("telemetry")) {
             logReader->add_source(new MetadataReader(*logReader, vm["telemetry"].as<string>()));
-            currentState.hasMetadataSource = true;
+            newState.hasMetadataSource = true;
+            cout << "Adding Telemetry source..." << endl;
         }
 
         if (vm.count("addr") && vm.count("port")) {
             logReader->add_source(new MetadataReader(*logReader, vm["addr"].as<string>(), vm["port"].as<string>()));
             newState.hasMetadataSource = true;
+            cout << "Adding Telemetry source..." << endl;
         }
 
 #ifdef HAS_DECKLINK
         if (vm.count("decklink")) {
-            importer.add_source(new DeckLinkImport(logReader), 500);
+            importer.add_source(new DeckLinkImport(logReader, goProRect), 500);
```

Comment:
This is hardcoded to imply that the decklink capture device will be the rectilinear go pro.

-Shouldn't it be configurable via CLI?